scripts/any/8fef1e9/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: generic
+sudo: false
```

Comment:
Remove this `sudo:false`. Otherwise the inheritance from the main .travis.yml will confuse travis and it ends up building for OS X as well: https://travis-ci.org/mapbox/mason/builds/218571809.

I'm a little fuzzy on why this happens but my sense is it is because the main `.travis.yml` has `sudo:false` inside a matrix. And I think the global one here conflicts with the one inside the matrix at https://github.com/mapbox/mason/blob/d8860aa211c9d41f0fddc9892d29c5b4c9588a32/.travis.yml#L9. Not sure if this is just the way yaml works or a travis bug. But staying away from globals inside the package specific `.travis.yml` will avoid the problem.